src/test/java/ClosestStationTest.java

```diff
@@ -0,0 +1,39 @@
+import com.google.gson.Gson;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+public class ClosestStationTest
+{
+    @Test
+    public void getClosestStation() throws IOException
+    {
+        Gson gson = new Gson();
+        InputStream in = ClassLoader.getSystemResourceAsStream("SubwayLines.json");
+        InputStreamReader reader = new InputStreamReader(in);
+        InputStream inStation = ClassLoader.getSystemResourceAsStream("Subwaystations.json");
+        InputStreamReader readerStation = new InputStreamReader(inStation);
+        //when
+        LineInfo lineInfo = gson.fromJson(reader, LineInfo.class);
+        StationInfo stationInfo = gson.fromJson(readerStation, StationInfo.class);
+        readerStation.close();
+        reader.close();
+        List<StationInfo.Station> closestPath = new ClosestStation().getClosestStation(stationInfo.features.get(1), stationInfo.features.get(3),stationInfo, lineInfo);
+
+        //then
+        assertEquals(2, closestPath.get(0).properties.objectid);
```

Comment:
You should make sure that stationInfo.features.get(1).properties.objectid is 2

And do the same thing for the end